### PR TITLE
Promote quantity formatter to public

### PR DESCRIPTION
**Pull request by bsteinbk**
- update release tags and add significantly more documentation.

### CODE REVIEW

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -52,3 +52,21 @@ The [FloatingViewportContent]($appui-react) component has been added to support
 ## Font Workspaces
 
 It is now possible to store and load fonts from a Font Workspace. See [Fonts]($docs/learning/backend/Fonts.md) for more details.
+
```
Comment (pmconne) [inline docs/changehistory/NextVersion.md:55]: Add an announcement that the quantity formatter APIs have been promoted from beta. Describe what can be done with them. If samples, sandboxes, or example code exist, link to them.

**Review by pmconne — CHANGES_REQUESTED** [review]
No `@public` APIs should be undocumented.

**Review by christophermlawson — APPROVED** [review]

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (bsteinbk) [reply docs/changehistory/NextVersion.md:55]: Added info and linked to learning doc.

**Comment by bsteinbk** [comment]
> No `@public` APIs should be undocumented.

All public and protected methods in QuantityFormatter now have documentation.

File: common/api/core-frontend.api.md [hunk 2]
```diff
@@ -7731,33 +7714,24 @@ export class QuantityFormatter implements UnitsProvider {
...
-    // (undocumented)
     protected _unitFormattingSettingsProvider: UnitFormattingSettingsProvider | undefined;
-    // (undocumented)
     get unitsProvider(): UnitsProvider;
     set unitsProvider(unitsProvider: UnitsProvider);
     }
 
-// @beta
+// @public
 export enum QuantityType {
```
Comment (pmconne) [inline common/api/core-frontend.api.md:7735]: These are all undocumented. What are the units of Angle - degrees? radians? Meters for area , volume, and length. Difference between the three "Length" types? Stationing?

**Review by pmconne — APPROVED** [review]

**Pull request merged**